recipes/ditaa/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "ditaa" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stathissideris/ditaa/releases/download/v0.11.0/ditaa-0.11.0-standalone.jar
+  sha256: 9418aa63ff6d89c5d2318396f59836e120e75bea7a5930c4d34aa10fe7a196a9
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  build:
+  host:
+  run:
+    - openjdk
+    - python
+
+test:
+  commands:
+    - ditaa --help
+
+about:
+  home: https://github.com/stathissideris/ditaa
+  license: LGPL-3.0-only
+  license_family: LGPL
+  license_file: LICENSE
```

Comment:
For java files, you should also list all third-party notices. This probably works better if you build the JAR from source.

Reply:
Updated. Now jar file is created from source.

Reply:
You still need to list all licenses in a Notice file.

Reply:
I don't think this has happened yet @ajaypnair09

Reply:
Also, I think you need to include the actual GPL license for something that is LGPL licensed.